card-style.cls

```diff
@@ -36,6 +37,27 @@
 
 \newcounter{cards}
 
+\newcommand{\CardType}[1]{%
+  \def\cardcolor{}
+  \ifnum\pdfstrcmp{#1}{Item}=0      \def\cardcolor{blue} \fi
```

Comment:
somehow I was assuming we distinguish between permanent and consumable items

Reply:
🤷